lang/cs/Org.Apache.REEF.IO.Tests/TestAzureDataLakeFileSystem.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Microsoft.Azure.DataLake.Store;
+using Microsoft.Azure.DataLake.Store.MockAdlsFileSystem;
+using Org.Apache.REEF.IO.FileSystem;
+using Org.Apache.REEF.IO.FileSystem.AzureDataLake;
+using Org.Apache.REEF.Tang.Implementations.Tang;
+using Xunit;
+
+namespace Org.Apache.REEF.IO.Tests
+{
+    public sealed class TestAzureDataLakeFileSystem
+    {
+        private readonly static Uri FakeUri = new Uri("http://fake.com");
```

Comment:
Even though it's fake data, the URI should match the expected format of an ADLS URI.